### ISSUE TITLE
Node status is still updated every  10s(nodeStatusUpdateFrequency) after enabling feature NodeLease

### ISSUE THREAD

**Issue by LiuNingqian** [kubernetes/kubernetes#110816]
### What happened?

After enabling feature gate NodeLease in kubelet, apiserver and controller-manager, namespace kube-node-lease is created and API resource lease related to each node behave normally. However, I find that field node.status is also updated every 10s(nodeStatusUpdateFrequency).

Related source code: https://github.com/kubernetes/kubernetes/blob/721bfa751924da8d1680787490c54b9179b1fed0/pkg/kubelet/kubelet_node_status.go#L425
https://github.com/kubernetes/kubernetes/blob/721bfa751924da8d1680787490c54b9179b1fed0/pkg/kubelet/kubelet_node_status.go#L594

I find that func nodeStatusHasChanged() always return true, the reason is apiequality.Semantic.DeepEqual()'s returning true.
I compare originalStatus and status and find that the order of image list is different between originalStatus.Images and status.Images.

The final reason is status.Images only be sorted by image size. When there are several same-sized images, the order is not always the same. The related code:
https://github.com/kubernetes/kubernetes/blob/721bfa751924da8d1680787490c54b9179b1fed0/pkg/kubelet/images/image_gc_manager.go#L130

Kubernetes version:
`Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.3", GitCommit:"721bfa751924da8d1680787490c54b9179b1fed0", GitTreeState:"clean", BuildDate:"2019-02-01T20:08:12Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.3", GitCommit:"721bfa751924da8d1680787490c54b9179b1fed0", GitTreeState:"archive", BuildDate:"2021-04-29T05:06:36Z", GoVersion:"go1.12.17", Compiler:"gc", Platform:"linux/amd64"}`

There are many same-sized images on our cluster node, and I want to know how to avoid the problem.

### What did you expect to happen?

Node.Status is updated every time duration nodeStatusReportFrequency.
The status.Images remain same for each query operation.

### How can we reproduce it (as minimally and precisely as possible)?

Pull as many as 10 same-sized images on one node, and observe the field lastHeartbeatTime of node.status.conditions.

### Kubernetes version

<details>

```console
$ kubectl version
# paste output here
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.3", GitCommit:"721bfa751924da8d1680787490c54b9179b1fed0", GitTreeState:"clean", BuildDate:"2019-02-01T20:08:12Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.3", GitCommit:"721bfa751924da8d1680787490c54b9179b1fed0", GitTreeState:"archive", BuildDate:"2021-04-29T05:06:36Z", GoVersion:"go1.12.17", Compiler:"gc", Platform:"linux/amd64"}
```

</details>

### Cloud provider

<details>
no
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by 249043822** [comment]
Hi,  this probem has been fixed in v1.16 by this PR:https://github.com/kubernetes/kubernetes/pull/79578, since your kube version v1.13 has not been maintained again, you can merge this pr to your private branch

/close

**Comment by LiuNingqian** [comment]
> Hi, this probem has been fixed in v1.16 by this PR:#79578, since your kube version v1.13 has not been maintained again, you can merge this pr to your private branch
> 
> /close

Okay, thank you so much.
